readme.txt

```diff
@@ -0,0 +1,1414 @@
+=== WP REST API (WP API) ===
+Contributors: rmccue, rachelbaker
+Tags: json, rest, api, rest-api
+Requires at least: 3.9
+Tested up to: 4.4
+Stable tag: 1.2.4
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+Access your site's data through an easy-to-use HTTP REST API (Version 1, deprecated).
```

Comment:
Couple of things:

1. We aren't tracking the `readme.txt` in the Github repo, so this shouldn't be merged to master.
2. We need to be careful about the language we're using, as (I don't believe) we recommend using v2 on production sites. If you need a production REST API, you should use v1.